README.md

```diff
@@ -338,6 +338,7 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 * [emitter](https://emitter.io/) - Free open source real-time messaging service that connects all devices. This publish-subscribe messaging API is built for speed and security.
 * [EventStore](https://github.com/EventStore/EventStore) - The open-source, functional database with Complex Event Processing in JavaScript. [https://geteventstore.com](https://geteventstore.com)
 * [Foundatio](https://github.com/exceptionless/Foundatio#queues) - A common interface with in memory, redis and azure implementations.
+* [GnatMQ](https://github.com/gnatmq/gnatmq) - Embedded MQTT broker/server, open-source and 100% implemented in .Net
```

Comment:
@VladimirAkopyan so this project not support asp.net core ?

Reply:
It does support .Net Core, .Net Standard, Linux and UWP
It's not related to ASP or web things

Reply:
@ngohungphuc I just have a look at https://github.com/gnatmq/gnatmq/tree/IntegrationAPI/GnatMQ. Actually, they are working for .NET Core version there. I think we just keep it `OPEN` until they merge it to `master` branch?

Reply:
Agree